Answers.md

```diff
@@ -1,3 +1,22 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
+1. Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?  
+>A docker container is different from a virtual machine because it is leaner and specialized for a much smaller range of tasks. It is more aptly  
+>suited for portability because it takes up far fewer resources to run.
+
+
+2. Given the commend `docker run -p 49160:8080 -d <your_docker_username>/<your_docker_image_name>`, what does `49160:8080` specify?  
+>49160:8080 specificies that 49160 will be how someone can reach the image of the item that was hosted on localhost:8080 in your docker image
+
+3. What is the main purpose of using a "container orchestration platform" such as Kubernetes or Docker Swarm?  
+>  These tools limit the amount of containers and resources being used according to the demand
+
+4. How do you change the number of replicas in a Kubernetes cluster?  
+>
```

Comment:
In order to scale a Kubernetes cluster you need to edit the config files and update the `replicas` value. Once you've done that, run the `apply` command from kubernetes cli with the updated file.